client/src/components/WordCloudModal/GenerateWordCloudValues.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Description: React word cloud takes a list of json objects
+ *              as input. This code generates the list by 
+ *              getting search results from the DOM
+ *              and counting instances of individual or hyphenated words
+ *              using a dictionary mapping approach for efficiency.   
+ *              Un-interesting words are filtered from the word cloud.
+ * Date: 7/28
+ * Author: Rob Marcus
+ */
+
+// Set of invalid word cloud words/strings. 
+const INVALID_WORDS = new Set(['',
+                              'a',
+                              'and',
+                              'at',
+                              'for',
+                              'i',
+                              'in',
+                              'is',
+                              'it',
+                              'its',
+                              'of',
+                              'on',
+                              'the',
+                              'this',
+                              'to',
+                              '|']);
```

Comment:
How did you decide on these words in particular? You have "this" but not "that",  "its" but not "it's", and some conjunctions but not others.

Reply:
Experimentally looked at the word cloud and took notes on what words were polluting the chart basically. Have added a clarifying comment.